### ISSUE TITLE
🐞 Bug report: [IconBox Widget] Accessibility issue, 2 links are confusing for disabled users. Need only one.

### ISSUE THREAD

**Issue by Mulli** [elementor/elementor#17741]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Accesibility issue on [IconBox Widget]: Elementor generates 2 'a' links - that is confusing for disabled users.
Soolution: have a single link that wrap both: SVG icon AND title below

### Steps to reproduce

1. Set [IconBox Widget] WITH a link. on any Elementor release

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

Its a design coding issue with Elementor.
System info is irrelevant

**Comment by rami-elementor** [comment]
As mentioned in #9565 , you can click on the "link settings", there you will see an "attributes" field. Add `aria-label|Screen reader text` or `title|screen reader text` to make the link accessible.

**Comment by Mulli** [comment]
Issue #9565 does NOT solve the problem of 2 identical and confusing links.
I am looking for something like the following:
`<a href="link" >
            <div icon .....> ... </div>
            <header ....> ... </header>
 </a>`

Instead of the current produced html.

**Comment by jessevoogt** [comment]
I agree with Mulli https://github.com/elementor/elementor/issues/17741#issuecomment-1047349057, the presence of two links instead of one is the issue, and is NOT solved by the ability to set an aria-label or any other attribute on the link.  Also, I would strongly advocate as much as it is possible for the output to be accessible "by default", and _not_ require editors to perform several more actions to make something accessible.  The suggested solution for Issue #9565 doesn't actually solve the redundancy issue, but even if it did it requires too many steps for something that should just be automatic.

As Mulli stated, the HTML should be modified to only have a single link.  However, block-level elements (such as divs) are not supposed to go inside of inline elements (such as <a> or <span>), so the HTML should ideally be something more like this:
```
<a class="elementor-icon-box-wrapper" href="#">
        <span class="elementor-icon-box-icon">
		<span class="elementor-icon">
			<i aria-hidden="true" class="far fa-building"></i>
                </span>
	</span>
        <span class="elementor-icon-box-content">
		<h3 class="elementor-icon-box-title">Custom Label</h3>
         </span>
</a>
```

**Comment by jessevoogt** [comment]
In case anyone else stumbles onto this, here's a (not yet thoroughly tested) band-aid solution (in which I'm attempting not to change overall HTML structure) that hides the empty link from screen readers and prevents it from being tab accessible - but please note that accessibility checkers like WAVE will still report the link as being an empty link, despite aria-hidden="true":

```
function fixElementorBoxWidgetAccessibility(){
	var $ = jQuery; 
	$('.elementor-icon-box-wrapper a.elementor-icon').each(function(){
		var label = $.trim($(this).text()) || $.trim($(this).attr('aria-label')) || $.trim($(this).attr('aria-labelledby'));
		if(!label) $(this).attr('tabIndex', '-1').attr('role', 'presentation').attr('aria-hidden', 'true')
	});
}
jQuery(document).on('ready ajaxComplete', fixElementorBoxWidgetAccessibility);
jQuery(window).on('load', fixElementorBoxWidgetAccessibility);
fixElementorBoxWidgetAccessibility();
```

**Comment by nicholaszein** [comment]
Hi @Mulli and others!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.16! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
